buf/registry/module/v1beta1/commit_service.proto

```diff
@@ -77,11 +77,12 @@ message ResolveCommitsRequest {
   // Once the resource is resolved, this reference is interpeted to refer to the following Commit:
   //   - If a Module is referenced, this is interpreted to mean the latest released Commit on the Module.
   //   - If a Commit is referenced, this just references that specific commit.
-  //   - If a Tag is referenced, this is interpreted to mean the Commit associated with the Tag.
+  //   - If a Tag is referenced, this is interpreted to mean the released Commit associated with the Tag.
+  //     Tags referencing unreleased commits cannot be resolved.
   //   - If a VCSCommit is referenced, this is interpreted to mean the Commit associated with the VCSCommit.
   //   - Is a Branch is referenced, this is interpreted to mean the latest Commit on the Branch.
   //   - If a Digest is referenced, this is interpreted to mean the latest released Commit that has this Digest.
-  //     Digests referencing unreleased Commits cannot be referenced.
```

Comment:
This was correct before

Reply:
This reads weird, though.
> Digests referencing unreleased Commits cannot be referenced.

... but I just referenced it. It _can_ reference an unreleased commit, but you cannot _resolve_ an unreleased commit.